diagnostics/app/model/diagnostics/DynamoDbReport.scala

```diff
@@ -72,7 +72,7 @@ object CSPDynamoDbReport extends DynamoDbReport[CSPReport] {
       .withAttributeUpdates(count.asJava)
 
     client.updateItemFuture(update) onFailure {
-      case error: Throwable => log.error(s"Failed to log CSP report to DynamoDB", error)
+      case error: Throwable => log.error(s"Failed to log CSP report to DynamoDB: $error", error)
```

Comment:
you can do that if you want to exclude the error object

`log.error` can also take two parameters, a string and a `Throwable`, which allows stack trace reporting as well as the message: https://github.com/playframework/playframework/blob/master/framework/src/play/src/main/scala/play/api/Logger.scala#L139